plugin/trino-accumulo/pom.xml

```diff
@@ -324,6 +324,7 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
nice catch

Can this be enforced?

Reply:
I looked around but couldn't find anything to enforce this.